frontend/src/actions/WindowActions.js

```diff
@@ -479,17 +479,21 @@ export function initWindow(windowType, docId, tabId, rowId = null, isAdvanced) {
           fetchAdvancedFields: isAdvanced,
         }).catch((e) => {
           dispatch(getWindowBreadcrumb(windowType));
-          dispatch(
-            initDataSuccess({
-              data: {},
-              docId: 'notfound',
-              includedTabsInfo: {},
-              scope: 'master',
-              saveStatus: { saved: true },
-              standardActions: [],
-              validStatus: {},
-            })
-          );
+          if (e.response.status === 404) {
```

Comment:
@petrican in the issue requirements, i was talking about when the document is loading first time.
Before getting the actual data (or layout?) i think somewhere we have hardcoded the default values to be something like not found....

Reply:
That is the only place where this is set for the document based on the description you gave. I've tried to replicate on slow instances like BM or SP but without success. I can dig deeper into this.

Reply:
@petrican  now i cannot reproduce it either. OK, let's keep an eye on it.

Reply:
@petrican  found it, it's in included tabs

![image](https://user-images.githubusercontent.com/1244701/150090528-e92175c2-eeb2-4544-af48-2a4362764ecb.png)


we need there a big rotating gear.

about the change you did here, i would revert it.

Reply:
the change is not merged. Yes I can revert it in my branch. But can you give me an instance where it happens ? i see it is on your localhost. Thanks

Reply:
Anywhere @petrican, you just have to throttle your network connection to make it more obvious...

Reply:
All right. Thanks! Will check